docs/site/sidebars/lb4_sidebar.yml

```diff
@@ -177,11 +177,19 @@ children:
         url: core-tutorial-part11.html
         output: 'web, pdf'
 
-- title: 'Usage Scenarios'
+- title: 'How-to guides'
   url: Usage-scenarios.html
   output: 'web, pdf'
   children:
 
+  - title: 'REST APIs'
```

Comment:
Discussion point: should we call this section "REST APIs" or something like "Building REST APIs"?

IMO, "REST APIs" could be interpreted as covering both building and consuming APIs, while this section of our docs is covering the building part only. For consumption, we already have the page "Calling other APIs and web services" that's outside of "REST APIs".

Reply:
Since it's from a LoopBack context, "REST APIs" in that particular location should be understood as configuring/building REST APIs.

Reply:
I'll go with "Building REST APIs" then.